### PR TITLE
Fix a missing function rename

### PR DESCRIPTION
**Pull request by nilsvu**
## Proposed changes

This is broken on develop, looks like a rebase conflict between #3691 and #3741 .

### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### CODE REVIEW

**Review by knelli2 — APPROVED** [review]
Oops yup looks like I missed one. Thanks for finding this!

**Review by nilsdeppe — DISMISSED** [review]

File: src/ApparentHorizons/StrahlkorperGr.cpp [hunk 1]
```diff
@@ -896,7 +896,7 @@ template <typename Frame>
 void radial_distance(const gsl::not_null<Scalar<DataVector>*> radial_distance,
                      const Strahlkorper<Frame>& strahlkorper_a,
                      const Strahlkorper<Frame>& strahlkorper_b) {
-  if (strahlkorper_a.center() != strahlkorper_b.center()) {
+  if (strahlkorper_a.expansion_center() != strahlkorper_b.expansion_center()) {
```
Comment (knelli2) [inline src/ApparentHorizons/StrahlkorperGr.cpp:899]: Oh actually you missed two a couple lines down in the error statement.

**Review by knelli2 — CHANGES_REQUESTED** [review]

**Review by nilsdeppe — APPROVED** [review]

**Review by knelli2 — APPROVED** [review]

**Review by geoffrey4444 — APPROVED** [review]

**Review by markscheel — APPROVED** [review]
Why didn't this fail compilation in the last PR?

**Comment by nilsvu** [comment]
Probably didn't rebase. No big deal, it's quick enough to fix

**Pull request merged**